src/adapters/streaming-key-ttl-checker.ts

```diff
@@ -0,0 +1,39 @@
+import { AppComponents } from '../types'
+import { IStreamingKeyChecker } from '../types/checker.type'
+import { CronJob } from 'cron'
+
+export async function createStreamingKeyTTLChecker(
+  components: Pick<AppComponents, 'logs' | 'sceneStreamAccessManager'>
+): Promise<IStreamingKeyChecker> {
+  const { logs, sceneStreamAccessManager } = components
+  const logger = logs.getLogger(`streaming-key-ttl-checker`)
+  let job: CronJob
+
+  async function start(): Promise<void> {
+    job = new CronJob(
+      '*/10 * * * *', // every 10 minutes
+      async function () {
+        try {
+          logger.info(`Looking into active streamings.`)
```

Comment:
Just to make the log look different than the other one we already have for the other job:
```suggestion
          logger.info(`Running job to remove expired streaming keys.`)
```